docs/user-guide/amor/amor-reduction.ipynb

```diff
@@ -18,6 +18,16 @@
     "## Setup"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import warnings\n",
+    "warnings.filterwarnings('ignore')"
```

Comment:
I still think we should not filter warnings in notebooks?

Reply:
This is already fixed on the main branch, https://github.com/scipp/essreflectometry/commit/9c09b1f6d34272592920e066d88d1c264f60a979.

When I try to merge this we will get a conflict and the ignore will go away